kvision-tools/kvision-gradle-plugin/src/main/kotlin/io/kvision/gradle/KVisionPlugin.kt

```diff
@@ -121,12 +121,10 @@ abstract class KVisionPlugin @Inject constructor(
             dependsOn(tasks.all.browserProductionWebpack)
         }
 
-        afterEvaluate {
```

Comment:
I don't think `afterEvaluate` is needed here. `tasks.all.processResources` returns a task collection that will live-update as matching tasks (in this case, those with the correct type), are added, and then `configureEach` will only configure them as each is required.

You might have discovered it's needed in more thorough testing though? :)

If so, maybe try replacing `.configureEach { }` with `.all { }`

That said -if `afterEvaluate` works, it works! It's fine as it is, I'm just proposing.

Reply:
This is not my `afterEvaluate` - you have added it :-)

Reply:
Haha no way. Okay... my mistake! I'm not sure what my thinking was then. I guess I copied it and didn't come back to look at it again.